templates/upload_success.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>CSV Upload Success</title>
+</head>
+<body>
+    <p>Accepted {{ num_shifts }} shifts from Mobilize CSV file {{ filename }}</p>
+    <a href="/mobilize/uploadcsv">Upload another CSV file</a>
```

Comment:
The href here should refer to `/docs/upload_csv.html`, as you took out this endpoint.